openapi/listings/listings.yaml

```diff
@@ -142,25 +164,56 @@ components:
               type: string
             syncingError:
               type: array
-              description: Any errors reported while publishing your information to the source
+              description: Any errors stopping the business information from being updated by the listing source.
               items:
                 type: object
                 properties:
                   errorCode:
-                    description: Use this to determine why a publishing listing information to a source has failed
+                    description: |-
+                      businessName
+                      streetAddress
+                      postalCode
+                      city
+                      wholeAddress - Error is not isolated to street address, city, etc.
+                      phoneNumber
+                      hoursOfOperation
+                      categories
+                      descriptionLong
+                      descriptionShort
+                      email
+                      paymentOptions
+                      photos
+                      website
+                      unhandledError
```

Comment:
From the partner's perspective, what is the difference between `unhandledError` and `unavailableReason`? 
Ideally, the error codes should correspond to actions the partner can take automatically instead of just the name of a field.

Reply:
I meant to remove unhandled error